src/library_fs.js

```diff
@@ -948,13 +952,7 @@ FS.staticInit();
     stat(path, dontFollow) {
       var lookup = FS.lookupPath(path, { follow: !dontFollow });
       var node = lookup.node;
-      if (!node) {
-        throw new FS.ErrnoError({{{ cDefs.ENOENT }}});
-      }
```

Comment:
I don't think `lookupPath` returns an `undefined` node ever, it would throw ENOENT itself so this check is not needed.